sf_stats.Api/sf_stats.Api/appsettings.Development.json

```diff
@@ -29,14 +29,14 @@
       {
         "Name": "MSSqlServer",
         "Args": {
-          "connectionString": "Server=CHILLTOP-W10;Database=SF_Stats;Trusted_Connection=True;",
+          "connectionString": "Server=JOSHLAPTOP\\SQLEXPRESS;Database=SF_Stats;Trusted_Connection=True;",
           "tableName": "Log"
         }
       }
     ]
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=CHILLTOP-W10;Database=SF_Stats;Trusted_Connection=True;"
+    "DefaultConnection": "Server=JOSHLAPTOP\\SQLEXPRESS;Database=SF_Stats;Trusted_Connection=True;"
   },
   "FeatureFlags": {
     "SQLVersion": "MSSQL"
```

Comment:
did we want to commit this?

Reply:
nope, I'll check out the main branch file